packages/typespec-go/test/armlargeinstance/zz_models.go

```diff
@@ -188,12 +188,12 @@ type NetworkProfile struct {
 
 // Operation - Details of a REST API operation, returned from the Resource Provider Operations API
 type Operation struct {
-	// Extensible enum. Indicates the action type. "Internal" refers to actions that are for internal only APIs.
-	ActionType *ActionType
-
-	// READ-ONLY; Localized display information for this particular operation.
+	// Localized display information for this particular operation.
 	Display *OperationDisplay
 
+	// READ-ONLY; Extensible enum. Indicates the action type. "Internal" refers to actions that are for internal only APIs.
```

Comment:
this change is from the typespec mgmt. spec change: https://github.com/Azure/typespec-azure/pull/1881/files#diff-823f10c1812ffa803f58da35c7d41623a79c2e5c37f0c15d09ce66bcfdcfbe97